python/tvm/contrib/rocm.py

```diff
@@ -0,0 +1,51 @@
+"""Utility for ROCm backend"""
+import sys
+import subprocess
+from . import util
+from ..api import register_func
+
+def rocm_link(in_file, out_file):
+    """Link relocatable ELF object to shared ELF object using lld
+
+    Parameters
+    ----------
+    in_file : str
+        Input file name (relocatable ELF object file)
+
+    out_file : str
+        Output file name (shared ELF object file)
+    """
+    args = "ld.lld -shared " + in_file + " -o " + out_file
+    proc = subprocess.Popen(
+        args, shell=True,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT)
+    (out, _) = proc.communicate()
+
+    if proc.returncode != 0:
+        sys.stderr.write("Linking error using ld.lld:\n")
```

Comment:
do raise instead, c.f. https://github.com/dmlc/tvm/blob/master/python/tvm/contrib/cc.py#L51 The nvcc one is a bit outdated and need to be updated